src/runtime_src/core/include/experimental/xrt-next.h

```diff
@@ -84,6 +84,11 @@ XCL_DRIVER_DLLESPEC
 int
 xclGetDebugProfileDeviceInfo(xclDeviceHandle handle, xclDebugProfileDeviceInfo* info);
 
+XCL_DRIVER_DLLESPEC
+int 
+xclGetSubdevPath(xclDeviceHandle handle,  const char* subdev,
+                 uint32_t idx, char* sysfsPath, size_t size);
+
```

Comment:
Please document.  
Also is this a public API?   This won't work on windows, I don't think it should be exposed in xrt-next.h

Reply:
removed, I found some other APIs such as 

xclCreateProfileResults(xclDeviceHandle, struct ProfileResults**);
xclGetProfileResults(xclDeviceHandle, struct ProfileResults*);
xclDestroyProfileResults(xclDeviceHandle, struct ProfileResults*);

it's only for XDP internal use, should we remove them from xrt-next.h?